tar/tarsnap.1-man.in

```diff
@@ -397,6 +397,10 @@ Allow descent into synthetic filesystems such as procfs.
 Normally archiving of such filesystems is a silly thing to do, hence the
 name of the option.
 .TP
+\fB\--iso-dates\fP
+(-t mode only)
+Prints file and directory dates as yyyy-mm-dd hh:mm:ss.
```

Comment:
Huh, I could have sworn that I checked somewhere before I started writing man entries in that style.  Will fix, as well as imperatizing "Creates and initializes the cachedir...", "Checks the configuration...", etc.